abl/robot/base.py

```diff
@@ -439,6 +439,7 @@ def run(self):
             if self.raise_exceptions:
                 raise
             self.error_handler.report_exception()
+            self.logger.exception("An unhandled exception occured.")
```

Comment:
This should be logged before re-raising the exception two lines up or you won't log for exception-raising robots.